spec/researches/buildKeywordFormsSpec.js

```diff
@@ -1,4 +1,4 @@
-const buildKeywordForms = require( "../../js/researches/buildKeywordForms.js" );
+const buildKeywordForms = require( "../../src/researches/buildKeywordForms.js" );
```

Comment:
I think we usually use the build files in the js folder for the specs.

Reply:
Fixed.